src/SectionsNavigation.Uno/MultiFrame.cs

```diff
@@ -297,7 +297,7 @@ public async Task RemoveFrame(string name)
 				switch (transitionInfo)
 				{
 					case DelegatingFrameSectionsTransitionInfo frameTransition:
-						await frameTransition.Run(previousFrame.Frame, nextFrame.Frame, true);
```

Comment:
we should probably add the prefix before the boolean value because I had to search in the code what this was for :P

Reply:
Done. I did it also for the frame, since it was also confusing for me at first